resources/keycloak/custom-user-provider/src/main/java/com/cvmanager/auth/provider/user/pojos/UserObject.java

```diff
@@ -0,0 +1,147 @@
+
+    @JsonProperty("keycloak_id")
+    private String id;
+    @JsonProperty("user_id")
+    private Integer user_id;
+    @JsonProperty("email")
+    private String email;
+    @JsonProperty("first_name")
+    private String firstName;
+    @JsonProperty("last_name")
+    private String lastName;
+    @JsonProperty("created_timestamp")
+    private Long createdTimestamp;
+    @JsonProperty("super_user")
+    private Integer superUser;
+    @JsonProperty("organizations")
+    private List<OrganizationObject> organizations;
+
+    public static UserObject fromJoinedResultSet(ResultSet re) {
+        UserObject user = new UserObject();
+        try {
+            user.setId(re.getString("keycloak_id"));
+            user.setUserId(re.getInt("user_id"));
+            user.setEmail(re.getString("email"));
+            user.setFirstName(re.getString("first_name"));
+            user.setLastName(re.getString("last_name"));
+            user.setCreatedTimestamp(re.getLong("created_timestamp"));
+            user.setSuperUser(re.getInt("super_user"));
+            user.setOrganizations(OrganizationObject.listFromString(re.getString("organizations")));
+        } catch (Exception e) {
+            log.error("Error parsing UserObject from SQL Result: ", e);
+        }
+        return user;
+    }
+
+    public static UserObject fromResultSet(ResultSet re) {
+        UserObject user = new UserObject();
+        try {
+            user.setId(re.getString("keycloak_id"));
+            user.setUserId(re.getInt("user_id"));
+            user.setEmail(re.getString("email"));
+            user.setFirstName(re.getString("first_name"));
+            user.setLastName(re.getString("last_name"));
+            user.setCreatedTimestamp(re.getLong("created_timestamp"));
+            user.setSuperUser(re.getInt("super_user"));
+            user.setOrganizations(null);
+        } catch (Exception e) {
+            log.error("Error parsing UserObject from SQL Result: ", e);
+        }
+        return user;
+    }
+
+    public static Map<String, Object> toTokenMap(UserModel user) {
+        // Generate Map from user object, for use in custom token mapper. This only includes fields which are not already present in the token data.
+        // - user_created_timestamp
+        // - super_user
+        // - organizations
+        //     - org
+        //     - role
+        Map<String, Object> map = new HashMap<String, Object>();
+        map.put("user_created_timestamp", user.getCreatedTimestamp());
+        map.put("super_user", user.getFirstAttribute("super_user"));
+        map.put("organizations", OrganizationObject.mapListFromString(user.getFirstAttribute("organizations")));
+        return map;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public Integer getUserId() {
+        return user_id;
+    }
+
+    public void setUserId(Integer user_id) {
```

Comment:
suggestion(non-blocking): the same as the field name suggestion above
```suggestion
    public void setUserId(Integer userId) {
```

Reply:
updated